actors/actor.go

```diff
@@ -57,8 +57,7 @@ func (p *ActorParser) GetMetadata(txType string, msg *parser.LotusMessage, mainM
 		metadata, err = p.ParseReward(txType, msg, msgRct)
 	case manifest.VerifregKey:
 		metadata, err = p.ParseVerifiedRegistry(txType, msg, msgRct)
-	case manifest.EvmKey:
-		metadata, err = p.ParseEvm(txType, msg, mainMsgCid, msgRct, ethLogs)
```

Comment:
I don't think we have to remove this. We could just remove the ethLogs argument, as it is not necessary anymore. However, we still want to parse metadata for evm txs.